contracts/test/integration/MultiSubnet.t.sol

```diff
@@ -49,12 +54,16 @@ contract MultiSubnetTest is Test, IntegrationTestBase {
     RootSubnetDefinition public rootSubnet;
     TestSubnetDefinition public nativeSubnet;
     TestSubnetDefinition public tokenSubnet;
+    TestSubnetDefinition public deflationaryTokenSubnet;
+    TestSubnetDefinition public inflationaryTokenSubnet;
+    TestSubnetDefinition public nilTokenSubnet;
 
     IERC20 public token;
+    IERC20 public deflationaryToken;
+    IERC20 public inflationaryToken;
+    IERC20 public nilToken;
```

Comment:
Do we need these as state variables? We only use each in its own test, once, so I think each one of these could be local variables in their corresponding test case.

Reply:
@snissn ^^

Reply:
I think that we should put these variables in the setUp in case we want to re-create more tests around these three tokens. Also the setUp requires a few variables that aren't saved as state, so we would need to place rootGateway and rootSubnetName into state at least in order to set up token subnets for them calling `createTokenSubnet`.